test/Microsoft.ServiceFabric.Services.Remoting.Tests/ExceptionSerializerTest.cs

```diff
@@ -82,7 +82,10 @@ public void UsesDefaultConvertorsIfNullPassed()
             [Fact]
             public void UsesPassedRemotingListenerSettings()
             {
-                var expectedSettings = new FabricTransportRemotingListenerSettings();
+                var expectedSettings = new FabricTransportRemotingListenerSettings()
+                {
+                    RemotingExceptionDepth = 17
```

Comment:
Hey @gobradovic, why do we need to set the `RemotingExceptionDepth`? I remember removing it because this test uses `Assert.Same`, which verifies object reference equality, so it shouldn't matter what the contents of that object are.

Reply:
@olegsych Ah yes, I noticed this while I was porting some changes and I thought I deleted it by mistake. Didn't look too deeply into it. Discarding this PR.